.applier/templates/omp-frontend-buildconfig.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: omp-frontend
+objects:
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: omp-frontend
+    name: omp-frontend
+  spec:
+    failedBuildsHistoryLimit: 5
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: omp-frontend:latest
+    postCommit: {}
+    resources: {}
+    runPolicy: Serial
+    source:
+      git:
+        ref: master
+        uri: "${GIT_URL}"
+      type: Git
+    strategy:
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: nodejs:10
```

Comment:
Would be good to param this. Ideally run this on Apache or nginx

Reply:
Note that changing to Apache or Nginx will require a change in how the frontend is actually serving content. As it currently stands, the app is using server-side rendering in JavaScript to try to improve client experience (and honestly as a PoC for how we might want to do this in the future), but we can switch it to a static content mode and use one of those servers if desired.
I'll tackle other suggested changes here shortly.